src/messages/MMgrReport.h

```diff
@@ -36,7 +36,7 @@ class PerfCounterType
     ::encode(path, bl);
     ::encode(description, bl);
     ::encode(nick, bl);
-    assert(type < 256);
+    assert(sizeof(type) == 1);  // enum type declared as uint8_t
```

Comment:
could be
```c++
static_assert (sizeof(type) == 1 , "enum type declared as uint8_t");
```

```c++
static_assert ( bool_constexpr )	//	(since C++17)
```